config/envs/development.py

```diff
@@ -20,3 +20,12 @@ class DevelopmentConfig(Config):
     TEMP_FILE_PATH = Path("app") / "export_config" / "output"
 
     GENERATE_LOCAL_CONFIG = True
+
+    DEBUG_USER_ON = True
+    DEBUG_USER = {
```

Comment:
in local do we need to put these details before login or it will create dummy auth cookie for the session ?

Reply:
Hey Nuwan, so if running locally with `DEBUG_USER` config as above, you will be able to bypass auth and access e.g., https://fund-application-builder.levellingup.gov.localhost:3011/dashboard without a problem

The way `DEBUG_USER` works doesn't create a dummy cookie, you just hit an `HTTPException` with every single request but keep getting allowed through owing to the workaround. For quick reference, code [here](https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/authentication/decorators.py#L109)